### PR TITLE
Add new feature release, allow deep links to contribute

### PR DESCRIPTION
**Pull request by mrjones-plip**

### CODE REVIEW

**Comment by mrjones-plip** [comment]
Sooo many reviewers!  Too many likely.  However, this is more a "yes, this happened and I know about it" type of review. Only review/comment on content if you feel the need!

File: content/en/core/releases/feature_releases.md
Comment (garethbowen) [inline content/en/core/releases/feature_releases.md:14]: I think Care should be Care Teams but if this has officially changed then ignore me!

```suggestion
The Product Team has two dedicated sub-teams (Allies and Care Teams) who try to deliver much requested features at a pace that is faster than our release cycle. To this end, select partners working directly with Product Team can access what are called "Feature Releases". 
```

File: content/en/core/releases/feature_releases.md
Comment (garethbowen) [inline content/en/core/releases/feature_releases.md:18]: Typo, but also, the next minor would be "3.11.0" not "3.10.2" which is the patch/service pack release.

```suggestion
Each Feature Releases will be a production ready branch of the most recent release of the CHT. The release will have undergone applicable discretionary testing. You can easily identify a Feature Release by its version which follows the pattern of `LATEST_RELEASE-FR-FEATURE`. For example, if the latest release is `3.10.1` then the Feature Releases for speedier upgrades would be `3.10.1-FR-speedier-upgrades`. When this feature is found to be of use, it would be automatically included in `3.11.0`, the next minor release after `3.10.1`.
```

File: content/en/core/releases/feature_releases.md
Comment (garethbowen) [inline content/en/core/releases/feature_releases.md:22]: I don't think we can guarantee easy upgrade as the next minor might be a major, and they can't upgrade to the next patch because they would lose the feature. What about...

```suggestion
Feature releases are always built on the latest version of the CHT. Once the feature is ready, it will be included in a regular CHT release and the project will be upgraded as soon as practical to get back on to a fully supported release.
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Thanks for writing this up!

File: content/en/core/releases/feature_releases.md
Comment (mrjones-plip) [reply content/en/core/releases/feature_releases.md:14]: agreed!

File: content/en/core/releases/feature_releases.md
Comment (mrjones-plip) [reply content/en/core/releases/feature_releases.md:18]: ah - right right - good catch!

File: content/en/core/releases/feature_releases.md
Comment (mrjones-plip) [reply content/en/core/releases/feature_releases.md:22]: works for me

**Comment by mrjones-plip** [comment]
Thanks @garethbowen  for the accuracy and typo fixes! Much obliged. 

over to the rest of the reviewers in case they have any concerns!

**Review by craig-landry — CHANGES_REQUESTED** [review]

File: content/en/core/releases/feature_releases.md
Comment (mrjones-plip) [reply content/en/core/releases/feature_releases.md]: I think this was actually @garethbowen's wording ;)  Teamwork FTW!

**Review by craig-landry — APPROVED** [review]

File: content/en/core/releases/feature_releases.md
Comment (abbyad) [inline content/en/core/releases/feature_releases.md:18]: Some suggestions:
- Italicize the first use of _Feature Release_, as [per style guide](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#italics-to-define-or-introduce-new-terms) ([and APA](https://apastyle.apa.org/style-grammar-guidelines/italics-quotations/italics))
- It's not clear what "applicable discretionary testing" means. It sounds like a fancy way of saying we'll do the testing we think is necessary... can we find a simpler way to say that, or specify what testing will be done? I've made a suggestion with simpler language, feel free to improve.
- I thought the pattern for version could be clearer by showing the variable parts.
- As discussed on call, we don't know by which version a feature will be ready to be introduced. I've simplified the text, but open to other suggestions.

```suggestion
Each _Feature Release_ is a production-ready release based on a recent release of the CHT that has gone through the necessary testing. You can easily identify a Feature Release by its version, which follows the pattern of `{BASE-RELEASE-NUMBER}-FR-{FEATURE-NAME}`. For example, if the latest release is `3.10.1` then the Feature Release for speedier upgrades would be `3.10.1-FR-speedier-upgrades`. When this feature is found to be successful and ready for wider distribution it will be included in the next release.
```

File: content/en/core/releases/feature_releases.md
Comment (abbyad) [inline content/en/core/releases/feature_releases.md:14]: Given the broader audience of the docs site this intro needs some more context, and here's a suggestion that may provide higher level intro without getting into internal processes.

```suggestion
To build and iterate on new features at a pace that is faster than our regular release cycle, some features are released in a _Feature Release_. Feature Releases are based on the most recent release and only include improvements related to a feature being developed. These releases are tested to be production-ready so that new features can be studied with CHT partners in a live deployment, with the aim of getting the feature ready for wider use in an upcoming release.
```

**Review by abbyad — CHANGES_REQUESTED** [review]
Nice to have this included! See line comments for notes.

File: content/en/core/releases/feature_releases.md
Comment (abbyad) [inline content/en/core/releases/feature_releases.md:5]: Is the `no_list` needed here?

File: content/en/core/releases/feature_releases.md
Comment (mrjones-plip) [reply content/en/core/releases/feature_releases.md:17]: Yeah - looks good - thanks for the input!  

That said @craig-landry really liked the phrase "discretionary testing".  Are you OK with @abbyad's suggestion to remove it?

Comment (craig-landry) [reply content/en/core/releases/feature_releases.md:17]: Yep, I agree that "discretionary testing" is another way of saying "we're going to test it as we see fit"
I liked "discretionary" since it's a short way to say "we'll do the testing we think is necessary". I'm cool with alternatives, I just haven't heard anything I liked better than that. I'm probably fine with whatever other suggestions people have though.

To the point of a suggestion, I don't see it here as part of this suggestion though; instead it looks to be omitted. Was that the intention or am I missing it somewhere else?

Comment (mrjones-plip) [reply content/en/core/releases/feature_releases.md:17]: chatted w/ @craig-landry  - we're good to go as is - but acknowledging we might revisit to add explicit wording around how much we test.

**Comment by mrjones-plip** [comment]
sorry @abbyad ! I commented and resolved the testing comment but didn't accept your suggestion. This was just resolved with [1ce5321](https://github.com/medic/cht-docs/pull/639/commits/1ce532138d55b3318853169ce11738eda02b4bc4).  We're good to go once you approve the PR.

**Review by abbyad — APPROVED** [review]
Looks great, thanks for pushing this through!

**Pull request merged**
